content/platform/hpe-alletra/home.md

```diff
@@ -0,0 +1,57 @@
+---
+title: HPE Alletra
+version: v0.1
+description: A paradigm shift in data infrastructure
+image: /img/storage-192.png
+width: large
+priority: 2
+tags:
+  - hpe-alletra
+  - hpe-alletra-6000
+  - hpe-alletra-9000
+---
+Shed the complexity and silos inherent in conventional hybrid cloud environments with category-defining cloud-native data infrastructure that delivers a cloud operating and consumption experience wherever data lives. Stop managing infrastructure — and start simply accessing and utilizing it, as a service and on demand.
+
+- Get the agility of cloud — everywhere
+- Run any app — without compromise
+- Free your data across hybrid cloud
+
+# Projects
+
+Plugins, SDKs and documentation.
+
+## [CSI Driver ![](Github)](https://github.com/hpe-storage/csi-driver)
+
+A Container Storage Interface (CSI) Driver for Kubernetes. The HPE CSI Driver for Kubernetes allows you to use a Container Storage Provider to perform data management operations on storage resources.
+
+- [View the API documentation](https://developer.hpe.com/api/hpe-nimble-csp/)
+
+- [Helm Chart](https://artifacthub.io/packages/helm/hpe-storage/hpe-csi-driver)
+
+- [Operator for Kubernetes](https://artifacthub.io/packages/olm/community-operators/hpe-csi-operator)
+
+- [Operator for OpenShift](https://access.redhat.com/containers/#/registry.connect.redhat.com/hpestorage/csi-driver-operator)
+
+- [Visit documentation on SCOD](https://scod.hpedev.io/csi_driver/index.html)
+
+## [Storage Container Orchestrator Documentation ![](Github)](https://github.com/hpe-storage/scod)
+
+The definitive source for end-user documentation using Kubernetes and neighboring partner ecosystems with HPE Alletra.
+
+- [Explore the SCOD portal](https://scod.hpedev.io/)
+
+# Workshops-on-Demand
+
+Take advantage of our free, Jupyter-Notebook based Workshops-on-Demand available in the [Hack Shack](https://hackshack.hpedev.io/). These technical workshops provide you with an in-depth, hands-on learning experience where you can interact with and learn from the experts. Designed to fit your schedule, these workshops are available 24/7 – any time, from anywhere. A CSI workshop for HPE Alletra is available today.
+
+<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+<div class="w3-container w3-center w3-margin-bottom">
+  <a href="https://hackshack.hpedev.io/workshops"><button type="button" class="w3-btn w3-teal">Try now!</button></a>
+</div>
+
+---
+
+## Any questions on HPE Alletra?
+
+Join the [HPEDEV Slack Workspace](https://slack.hpedev.io/) and start a discussion in the [\#alletra](https://hpedev.slack.com/) channel.
```

Comment:
Hi Michael,
I have created the #alletra channel in HPE DEV workspace: 
https://hpedev.slack.com/archives/C025D75HHGC

You may want to set the Hotlink for #alletra accordingly.

Reply:
Done!